docs/sources/developers/plugins/publish-a-plugin.md

```diff
@@ -1,161 +1,59 @@
-  Alternative methods, such as installing private or development plugins on local Grafana instances, are available as per the guidelines provided in [this guide]({{< relref "../../administration/plugin-management#install-plugin-on-local-grafana" >}}).
-
-**Can I still use Angular for my plugin?**
-
-- No. We will not accept any new plugin submissions written in angular.
-  Please take a look at our [angular support deprecation documentation]({{< relref "../angular_deprecation" >}}).
-
-### Publishing your plugin for the first time
+To speed up the time it takes to review your plugin:
 
-**Do plugin signatures expire?**
+- Check that your plugin is ready for review using the [plugin validator](https://github.com/grafana/plugin-validator).
+- Read our [6 tips for improving your Grafana plugin before you publish](/blog/2021/01/21/6-tips-for-improving-your-grafana-plugin-before-you-publish/).
+- Refer to [plugin-examples](https://github.com/grafana/grafana-plugin-examples) to review best practices for building your plugin.
 
-- Plugin signatures do not currently expire.
+## Publish your plugin
 
-{{< figure src="/static/img/docs/plugins/plugins-submission-create2.png" class="docs-image--no-shadow" max-width="650px" >}}
+Follow these steps to publish your plugin for the first time.
 
 1. [Sign in](/auth/sign-in) to your Grafana Cloud account.
 1. In the left menu, under **Org settings**, click **My Plugins**.
-1. Click **Submit Plugin**.
+1. Click **Submit Plugin**. The Create Plugin Submission form appears.
+
+   {{< figure src="/static/img/docs/plugins/plugins-submission-create2.png" class="docs-image--no-shadow" max-width="650px" >}}
+
 1. Enter the information requested by the form.
    - **OS & Architecture:**
      - Select **Single** if your plugin archive contains binaries for multiple architectures.
      - Select **Multiple** if you'd like to submit separate plugin archives for each architecture.
        This can lead to faster downloads since users can select the specific architecture on which they want to install the plugin.
    - **URL:** A URL that points to a ZIP archive of your packaged plugin.
-   - **Source Code URL:** A URL that points to a public git repository or ZIP archive of your complete plugin source code.
+   - **Source Code URL:** A URL that points to a public Git repository or ZIP archive of your complete plugin source code.
    - **MD5:** The MD5 hash of the plugin specified by the **URL**.
    - The remaining questions help us determine the [signature level]({{< relref "./sign-a-plugin#plugin-signature-levels" >}}) for your plugin.
 1. Click **Submit**.
-   After you have submitted your plugin, we run an automated validation to make sure it adheres to our guidelines.
+   After you submit your plugin, we run an automated validation to make sure it adheres to our guidelines.
    Once your submission passes the validation, it's placed in a review queue.
```

Comment:
Restructured the sentence to clarify the antecedent for "it", otherwise "it" referred to the validation and not the submission. 
```suggestion
   Your submission is placed in the review queue once it passes the validation.
```